### PR TITLE
feat: organization avatar component

### PR DESCRIPTION
**Pull request by joeauyeung**
## What does this PR do?

	
This PR adds the `indicator` prop to the `Avatar` component and adds an `OrganizationAvatar` component.

Fixes # (issue)

## Requirement/Documentation

- If there is a requirement document, please, share it here.
- If there is ab UI/UX design document, please, share it here.

## Type of change

- [x] Chore (refactoring code, technical debt, workflow improvements)

## How should this be tested?

- Are there environment variables that should be set?
- What are the minimal test data to have?
- What is expected (happy path) to have (input and output)?
- Any other important info that could help to test that PR

## Mandatory Tasks

- [ ] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

## Checklist

- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes

### CODE REVIEW

File: packages/ui/components/avatar/Avatar.tsx [hunk 1]
```diff
@@ -66,27 +64,7 @@ export function Avatar(props: AvatarProps) {
...
-          <div
-            className={classNames(
-              "text-inverted absolute bottom-0 right-0 block rounded-full bg-green-400 ring-2 ring-white",
-              size === "lg" ? "h-5 w-5" : "h-2 w-2"
-            )}>
-            <div className="flex h-full items-center justify-center p-[2px]">
-              {size === "lg" && <Check />}
-            </div>
-          </div>
-        )}
```
Comment (joeauyeung) [inline packages/ui/components/avatar/Avatar.tsx:79]: I looked through the code base for `accepted={` and only found it being used by `AvatarGroup`. That was the only result so I'm assuming that this wasn't being used.

File: packages/features/ee/organizations/components/OrganizationAvatar.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+}: OrganizationAvatarProps) => {
+  return (
+    <Avatar
+      size={size}
+      imageSrc={imageSrc}
+      alt={alt}
+      indicator={
+        (organizationLogo || organizationName) && (
+          <div
+            className={classNames("absolute bottom-0 right-0 z-10", size === "lg" ? "h-5 w-5" : "h-10 w-10")}>
```
Comment (sean-brydon) [inline packages/features/ee/organizations/components/OrganizationAvatar.tsx:26]: I think the sizes are wrong here  - Pushing a storybook file change so we can easily test this also 
CleanShot 2023-08-16 at 09 48 09@2x

Comment (sean-brydon) [inline packages/features/ee/organizations/components/OrganizationAvatar.tsx:26]:
```suggestion
            className={classNames("absolute bottom-0 right-0 z-10", size === "lg" ? "h-3 w-3" : "h-10 w-10")}>
```

h-5 w-5 doesnt exist in tailwind

**Review by sean-brydon — APPROVED** [review]
Added a storybook component and change the sizes as w-5 and h-5 doesnt exist. 

We have some merge conflicts tho :/

**Pull request merged**